how to define functions

```diff
@@ -0,0 +1,62 @@
+vals = (3, 7)
+
+f x = (x, x + 1)
+
+g :: Int -> (Int, Int)
+g x = (x+1, x)
+
+myDouble  x = (if x > 145 then x else x*2) + 1  
+myDouble' x = if x > 100  
+                        then x  
+                        else x*2 
+
+main =
+  do
+    let (a, b) = vals
+    putStrLn (show a)
+    putStrLn (show b)
+
+    let (_, c) = f 4
+    putStrLn (show c)
+    
+    let (d, _) = g 4
+    putStrLn (show d)
+    
+    let y = myDouble 1
+    putStrLn (show y)
+    let y' = myDouble 146
+    putStrLn (show y')
+    let z = myDouble' 1
+    putStrLn (show z)
+    let z = myDouble' 146
+    putStrLn (show z')
+ 
+-- CURRYING:
```

Comment:
I do think we should talk about currying, but I'm wondering if it might be a little too much here - I'm thinking maybe we could leave this page as an *extremely gentle* introduction to functions, and then have a second page entitled "partial function application", what do you think?

Reply:
Yes, that makes sense. Should I add currying in another pull request or just leave it here?

Reply:
It's fine that you left it here; we might separate it out when we get ready to create the Phrasebook pages for them.